README.md

```diff
@@ -11,29 +11,70 @@ Tested with Python 3.11.4
 The following steps are not required but recommended. This will allow you to install packages in your isolated virtual environment instead of globally, reducing the risk of breaking system tools or other projects.
 
 1. Install [pyenv](https://github.com/pyenv/pyenv) and the [pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv) plugin.
-2. Download the appropriate Python version with `pyenv install 3.11.4` in the command line.
-3. Create a virtual environment with the appropriate Python version and name for your environment, for example `pyenv virtualenv 3.11.4 innovationserhebung`
-4. Activate the environment with `pyenv activate innovationserhebung`
+2. Download the appropriate Python version with 
+```shell
+pyenv install 3.11.4
+``` 
+in the command line.
+3. Create a virtual environment with the appropriate Python version and name for your environment, for example 
+```shell
+pyenv virtualenv 3.11.4 innovationserhebung
+```
+4. Activate the environment with 
+```shell
+pyenv activate innovationserhebung
+```
 
+### Install requirements
+
+Install the required libraries with the command line 
+```shell
+pip install -r requirements.txt
+```
 ### Run the APP
 
 1. Move to the `/app` directory.
-2. Install the required libraries with the command line `pip install -r requirements.txt`
-3. Run the server with the command line `uvicorn main:app --reload`
-4. Your terminal should show something like: 
+2. Run the server with the command line 
+```shell
+uvicorn main:app --reload
+```
+3. Your terminal should show something like: 
 
 ```
 INFO:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
 ```
 
+
+### Use tailwindcss and prettier-plugin-jinja-template
+
+1. If u want to work on the html templating files as well it would be useful to install tailwind, prettier and use the prettier-plugin-jinja-template as well.
+
+```shell
+npm install
+```
+
+
 ### Run tailwind watcher
 
 1. Move to the root directory
-2. Run `tailwindcss init`
-3. From now on you can start the tailwind watcher from the root directory via `tailwindcss -i app/styles/main.css -o app/static/css/main.css --watch`
+2. Run 
+```shell
+tailwindcss init
+```
```

Comment:
This is actually no longer needed, since tailwind is already configured in this repo. When I run it anyway, I get this message:

> tailwind.config.js already exists.